cognite/client/data_classes/workflows.py

```diff
@@ -1240,8 +1240,8 @@ class WorkflowTriggerDataModelingQuery(CogniteObject):
 
     def __init__(
         self,
-        with_: dict[str, ResultSetExpression] = {},
-        select: dict[str, Select] = {},
+        with_: dict[str, ResultSetExpression],
```

Comment:
Why with_, typo?

Reply:
`with`is a reserved python keyword.
therefore following the Query parameter naming in query.py